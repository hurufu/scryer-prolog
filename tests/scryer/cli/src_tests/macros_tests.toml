args = [
    "-f",
    "--no-add-history",
    "-g", "use_module(library(format)),listing(example/1)",
    "-g", "halt",
    "src/tests/macros.pl"
]
