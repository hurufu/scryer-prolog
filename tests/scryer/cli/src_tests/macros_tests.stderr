% Warning: Unknown macro fep # hello at line 91 of macros.pl
